src/Maestro/Maestro.Data/Migrations/BuildAssetRegistryContextModelSnapshot.cs

```diff
@@ -174,7 +174,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.HasIndex("SubscriptionId");
 
-                    b.ToTable("AssetFilter");
+                    b.ToTable("AssetFilters");
```

Comment:
The changes in this file are just re-ordered blocks but the diff looks horrible.